### PR TITLE
OSDOCS-2421: also describe SRE AWS access in STS mode

### PR DESCRIPTION
**Pull request by gregsheremeta**

### CODE REVIEW

**Review by fahlmant — APPROVED** [review]
lgtm

**Review by jonquilwilliams — COMMENTED** [review]
s/ For non-STS clusters, the `BYOCAdminAccess` role has the AdministratorAccess IAM policy attached and is used for administration. / For non-STS clusters, the `BYOCAdminAccess` role has the AdministratorAccess IAM policy attached, and this role is used for administration.

**Review by jonquilwilliams — COMMENTED** [review]
s/ For STS clusters, the `ManagedOpenShift-Support-Role` has the `ManagedOpenShift-Support-Access` policy attached and is used for administration. / For STS clusters, the `ManagedOpenShift-Support-Role` has the `ManagedOpenShift-Support-Access` policy attached, and this role is used for administration.

**Review by jonquilwilliams — COMMENTED** [review]
RH writing standards state that we must not use parentheses. So, I suggest changing this "(Both STS and non-STS clusters use the AWS STS service for SRE access.)" to a new paragraph that is subsumed under this section, and remove the parentheses.

**Review by arendej — APPROVED** [review]

**Comment by gregsheremeta** [comment]
/cc @pneedle-rh @EricPonvelle 
could someone take this over for me?

**Review by EricPonvelle — APPROVED** [review]
/lgtm

**Review by fahlmant — APPROVED** [review]
lgtm

File: modules/rosa-policy-identity-access-management.adoc [hunk 1]
```diff
@@ -43,7 +43,7 @@ Each of these access types have different levels of access to components:
 == SRE access to AWS accounts
 Red Hat personnel do not access AWS accounts in the course of routine {product-title} operations. For emergency troubleshooting purposes, the SREs have well-defined and auditable procedures to access cloud infrastructure accounts.
 
-SREs generate a short-lived AWS access token for the `BYOCAdminAccess` user using the AWS Security Token Service (STS). Access to the STS token is audit-logged and traceable back to individual users. The `BYOCAdminAccess` user has the AdministratorAccess IAM policy attached.
+SREs generate a short-lived AWS access token for a reserved role using the AWS Security Token Service (STS). Access to the STS token is audit-logged and traceable back to individual users. For non-STS clusters, the `BYOCAdminAccess` role has the AdministratorAccess IAM policy attached and is used for administration. For STS clusters, the `ManagedOpenShift-Support-Role` has the `ManagedOpenShift-Support-Access` policy attached and is used for administration. (Both STS and non-STS clusters use the AWS STS service for SRE access.)
```
Comment (bergerhoffer) [inline modules/rosa-policy-identity-access-management.adoc:46]: * `AdministratorAccess` should be in backticks (like the other role is)
* The parentheses around the last sentence should be removed. Also, would it make sense to put this sentence before the sentences that start with "For non-STS..." and "For STS..."?

**Review by bergerhoffer — COMMENTED** [review]
Just a few things!

**Review by arendej — APPROVED** [review]
lgtm

**Comment by gregsheremeta** [comment]
> s/ For non-STS clusters, the `BYOCAdminAccess` role has the AdministratorAccess IAM policy attached and is used for administration. / For non-STS clusters, the `BYOCAdminAccess` role has the AdministratorAccess IAM policy attached, and this role is used for administration.

done

**Comment by gregsheremeta** [comment]
> s/ For STS clusters, the `ManagedOpenShift-Support-Role` has the `ManagedOpenShift-Support-Access` policy attached and is used for administration. / For STS clusters, the `ManagedOpenShift-Support-Role` has the `ManagedOpenShift-Support-Access` policy attached, and this role is used for administration.

done

**Comment by gregsheremeta** [comment]
> RH writing standards state that we must not use parentheses. So, I suggest changing this "(Both STS and non-STS clusters use the AWS STS service for SRE access.)" to a new paragraph that is subsumed under this section, and remove the parentheses.

done

File: modules/rosa-policy-identity-access-management.adoc (hunk 1 above)
Comment (gregsheremeta) [reply modules/rosa-policy-identity-access-management.adoc]: done, all three

**Comment by gregsheremeta** [comment]
incorporated all suggestions, please re-review

**Review by EricPonvelle — APPROVED** [review]
/lgtm

**Comment by gregsheremeta** [comment]
rebased and squashed

**Pull request merged**